app/pages/adduser/controller.js

```diff
@@ -3,29 +3,32 @@ import { getContext } from './contextCreator';
 import { logger } from '../../logger';
 
 export const getAddUserContext = async ({ organisationId, accessToken }) => {
-  const options = { organisationId };
   // Need to call getOrgById to get org name
-  const orgData = await getData({ endpointLocator: 'getOrgById', options, accessToken });
-  logger.info(`Organisation ${organisationId} returned`);
-  return getContext(orgData);
+  const orgData = await getData({ endpointLocator: 'getOrgById', options: { organisationId }, accessToken });
+  if (orgData) {
+    logger.info(`Organisation ${organisationId} returned`);
+    return getContext(orgData);
+  }
+
+  throw new Error('No data returned');
 };
 
 export const postAddUser = async ({ organisationId, data, accessToken }) => {
-  logger.info(`Organisation ${organisationId} returned`);
   try {
     await postData({
       endpointLocator: 'postAddUser',
       options: { organisationId },
       body: data,
       accessToken,
     });
-    logger.info(`User ${JSON.stringify(data)} added`);
-    return { success: true };
+
+    logger.info(`User added: ${JSON.stringify(data)}`);
+    return { success: true, userAdded: `${data.firstName} ${data.lastName}` };
   } catch (err) {
-    if (err.response.status === 400) {
+    if (err.response && err.response.status === 400 && err.response.data) {
       return err.response.data;
     }
-    logger.error(err);
-    throw err;
+    logger.error(JSON.stringify(err));
```

Comment:
How about we add this logger line to the error handler so we always log the error.

Reply:
we need to have a whole discussion around error handling as it is inconsistent and even missing in places but I have removed this for now